scripts/jrelease.java

```diff
@@ -0,0 +1,371 @@
+                "Enter the git rev of the commit [RELEASE] Updated project version to " + nextReleaseVersion,
+                "xxxxxxx");
+        executeStep("git tag -a v" + nextReleaseVersion + " " + gitRevToTag + " -m \"" + nextReleaseVersion + " ("
+                + new SimpleDateFormat("yyyy-MM-dd").format(new java.util.Date()) + ")\"");
+        executeStep("git push " + gitUpstreamRemote + " v" + nextReleaseVersion);
+
+        // 13. https://ci.eclipse.org/jkube
+        printStep(13, "https://ci.eclipse.org/jkube");
+        // - Run build simplified-maven-central-release with tag as RELEASE_TAG build
+        // parameter
+        print("- Run build https://ci.eclipse.org/jkube/job/ReleasePipelines/job/simplified-maven-central-release/ with tag v"
+                + nextReleaseVersion + " as RELEASE_TAG build parameter");
+        print("Press enter to continue");
+        System.in.read();
+
+        // - Run simplified-downloads-eclipse-release with tag as RELEASE_TAG build
+        // parameter -> check https://download.eclipse.org/jkube/
+        // https://ci.eclipse.org/jkube/job/simplified-maven-central-release/
+        print("- Run https://ci.eclipse.org/jkube/job/ReleasePipelines/job/simplified-downloads-eclipse-release/ with tag v"
+                + nextReleaseVersion + " as RELEASE_TAG build parameter");
+        print("Press enter to continue");
+        System.in.read();
+
+        // 14. Wait for maven central (check
+        // https://repo1.maven.org/maven2/org/eclipse/jkube/kubernetes-maven-plugin-doc/
+        // )
+        printStep(14, "Wait for maven central, check");
+        print("- Check https://download.eclipse.org/jkube/");
+        print("- Check https://repo1.maven.org/maven2/org/eclipse/jkube/kubernetes-maven-plugin-doc/");
+        print("Press enter to continue");
+        System.in.read();
+
+        // 15. Create GitHub release from previously pushed tag to upstream, enable
+        // `discussions`
+        // Add changelog
+        // ./scripts/changelog.sh extract 1.13.1
+        printStep(15, "Create GitHub release from previously pushed tag to upstream, enable `discussions`");
+        print("Add changelog this to the release description:");
+        executeInteractiveCommand("bash", "-c", "./scripts/changelog.sh extract " + nextReleaseVersion);
+
+        // 16. Tag new release in https://github.com/jkubeio/jkube-website (keep empty
+        // title/description)
+        printStep(16, "Tag new release in https://github.com/jkubeio/jkube-website (keep empty title/description)");
+        print("Press enter to continue");
+        System.in.read();
+
+        // 17. Publish tweet in jkubeio profile
+        printStep(17, "Publish tweet in jkubeio profile");
+        // print the message to tweet:
+        // 🔥 @ECDTools #JKube new release 🚀
+        // 📦 v1.14.0
+        // ✨ Helidon support
+        // ✨ several improvements for helm
+        // ✨ CustomResource fragment improvements
```

Comment:
Do we want to keep these comments (these seem specific to previos release)?

Reply:
no you are right, I used them as a template.
Actually we could remove almost all the comments WDYT ?